edx_proctoring/templates/onboarding_exam/rejected.html

```diff
@@ -8,14 +8,32 @@
 {% endblock %}
 
 {% block body %}
-  <p>
-    {% blocktrans %}
-      Please contact
-      <a href="mailto:{{ integration_specific_email }}">
-        {{ integration_specific_email }}
-      </a> if you have questions, or to retry your onboarding exam.
-    {% endblocktrans %}
-  </p>
+
+  {% if experimental_procotoring_features %}
```

Comment:
What's the logic behind this condition to be true?

Reply:
This is set by the value of the CourseWaffle flag for rollout in LMS.